nemo_curator/modules/fuzzy_dedup/bucketstoedges.py

```diff
@@ -124,6 +124,12 @@ def buckets_to_edges(
         return result_df
```

Comment:
Alternatively this class since it follows the same `__call__` pattern can probably inherit from BaseModule?